src/bentoml/exceptions.py

```diff
@@ -107,8 +110,12 @@ class CLIException(BentoMLException):
 
 
 class YataiRESTApiClientError(BentoMLException):
+    """Raise when there is an error during push/pull bentos/models."""
+
     pass
 
 
 class ImportServiceError(BentoMLException):
```

Comment:
"failed to import the user's service file" probably is more accurate than "service.py".